### DISCUSSION TITLE
Bad move rate

### DISCUSSION THREAD

**Discussion by janbo208** [Equim-chan/mjai-reviewer#101] [Ideas]
Hello,
would it be possible to add a bad move rate/悪手率  metric to the review? For NAGA every move with less than 5% of recommendation level (推奨度) is counted as a bad move. https://twitter.com/NAGA025/status/1590570099022655488
I think this is more helpful compared to the raw match rate because not every mistake has the same impact.

**Comment by Equim-chan** [comment]
Note that Mortal is optimized for playing not reviewing (values other than the top move could be very off). It would be more useful and less misleading to specifically list these bad moves than to just give a number.

This may be a helpful feature but I don't have bandwidth to make it now.
